fdbclient/NativeAPI.actor.cpp

```diff
@@ -7604,7 +7656,15 @@ ACTOR Future<Standalone<VectorRef<BlobGranuleChunkRef>>> readBlobGranulesActor(
 		}
 
 		if (!self->trState->cx->blobWorker_interf.count(workerId)) {
-			Optional<Value> workerInterface = wait(self->get(blobWorkerListKeyFor(workerId)));
+			state Optional<Value> workerInterface;
+			if (self->getTenant().present()) {
+				// again, have to bypass system keys and tenant requirements
+				wait(store(
+				    workerInterface,
+				    getValue(self->trState, blobWorkerListKeyFor(workerId), self->getReadVersion(), UseTenant::False)));
```

Comment:
Does this logic work if the tenant isn't present? If so, we can do away with the if/else.